@here/harp-mapview/lib/MapView.ts

```diff
@@ -2125,9 +2158,9 @@ export class MapView extends THREE.EventDispatcher {
                 : Promise.resolve<Theme>(this.m_options.theme);
 
         themePromise.then((theme: Theme) => {
+            this.theme = theme;
```

Comment:
@zbigg @StefanDachwitz In the effects example, I want to use the `mapRenderingManager`'s API to apply effects in the code. But the theme also defines (or does not define) effects, so the API calls need to happen after the theme is loaded, but also after the theme is _applied_, hence this change. It seemed to me that the `ThemeLoaded` event is only used in `datasources/places` and that it does not break, so here is the change.

Thoughts?

Reply:
I think it's actually ok. Previous behavior was kind'a bug, because with event we stated that `theme (was) loaded` but in reality it was not yet applied.